docs/configuration.rst

```diff
@@ -53,44 +53,71 @@ The ``services`` section contains the definition of managed services by Cowbird.
 Each service is provided as a string that must match an actual implementation in Cowbird.
 Each service must be further configured with one or more of the following parameters. If
 a required parameter is missing for a service it will throw a ``ServiceConfigurationException`` exception.
+Additional parameters can be used for some services, such as ``admin_user`` and ``admin_password`` for a `Magpie`_
+service.
+
+Parameters :
+
+==============      ===========================================================================================
+Parameter name      Description
+==============      ===========================================================================================
+active:             [optional, default=False] Bool allowing to deactivate a service and stop managing it.
+priority:           [optional, default=math.inf] Relative priority between services while handling events.
+                    Lower values have higher priority, default value is last.
+url:                [optional, default=None] URI of the web service represented by this Cowbird service.
+                    Some Cowbird services do not represent web service, but other will throw an exception if
+                    missing.
+workspace_dir:      [optional, default=None] Location of the users workspace root.
+                    Required for the following services : ``FileSystem``, ``Catalog`` and ``Geoserver``.
```

Comment:
Remove `:` in parameter name and place them within double ``.

Create separate column for default value.